src/App.js

```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
-
+import Sidebar from './components/Sidebar/Sidebar';
```

Comment:
@morelmiles what naming convention are we using for the component folders/names? camelCase, PascalCase or some other case? I suggest we pick and stick to one.

Reply:
I agree

Reply:
Personally was using PascalCase